AnnoyancesFilter/sections/popups.txt

```diff
@@ -138,6 +138,8 @@ androidcentral.com##.article-shop-bar
 dpsrnext.com#$##myModal1 { display: none !important; }
 dpsrnext.com#$#.modal-backdrop { display: none !important; }
 dpsrnext.com#$#body { overflow: visible !important; padding-right: 0 !important; }
+buyagift.co.uk##body > div[style^="position: fixed; top: 0px; left: 0px;"]
```

Comment:
`buyagift.co.uk##.three` looks better

Reply:
@Sergey-Lyapin, pretty generic, though, there also `.two`, `.four` etc, they are used for z-index but still may be changed in the future.

Reply:
ok, maybe, but  I see that this class  is used for modal-mask only

Btw I like this rule more than yours:
`buyagift.co.uk##body > div[style*="position: fixed;"][style*="opacity: 0.8;"]`